### PR TITLE
Fix: Error with entry file paths beginning with ./

### PR DESCRIPTION
**Pull request by lammas**
Reference: issue #43

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
 .DS_Store
 node_modules
-dist
```
Comment (TrySound) [inline .gitignore]: Why did you do this? `dist` is published on npm, but github repo should be clrean.

**Comment by lammas** [comment]
Ah, damn. The second commit was not supposed to be part of the PR. Sorry.

**Comment by TrySound** [comment]
Also add please tests case for this.

**Comment by TrySound** [comment]
Cool! Do you know how to squash commits? Would be good to squash the first three.

**Comment by TrySound** [comment]
Thanks!
/cc @Rich-Harris

**Comment by TrySound** [comment]
@Rich-Harris Need to publish this fix. Can you share npm access?

**Comment by tivac** [comment]
Re-pinging @Rich-Harris since this continues to be an issue.

**Pull request merged**